controllers/groupsController.js

```diff
@@ -29,61 +29,71 @@ const summary = async (req, res) => {
 
   try {
     const { uid } = req.params;
```

Comment:
```suggestion
    const uid = req.params.uid;
```
변수에 할당할 데이터가 uid 하나 뿐이니까, 구조분해할당 방법을 사용하는 것보단 바로 변수로 할당하는 게 좋을 것 같아요!